src/main.tsx

```diff
@@ -5,11 +5,12 @@ import './index.css';
 import { Provider } from 'react-redux';
 import { store } from './stores/store.ts';
 import { Suspense } from 'react';
```

Comment:
Reorganize imports please. Imports from node_modules should be higher than custom ones